modules/auxiliary/admin/kerberos/forge_ticket.rb

```diff
@@ -82,6 +84,9 @@ def run
     when 'FORGE_GOLDEN'
       sname = ['krbtgt', datastore['DOMAIN'].upcase]
       flags = Rex::Proto::Kerberos::Model::TicketFlags.from_flags(golden_ticket_flags)
+    when 'DEBUG'
```

Comment:
Looks like I can't output a ticket without setting unrelated fields:

```
msf6 auxiliary(admin/kerberos/forge_ticket) > run ticket_path=./Administrator.ccache

[-] Msf::OptionValidateError The following options failed to validate: USER, DOMAIN, DOMAIN_SID
```

Reply:
yea that sucks, maybe this belongs in its own module? although that would feel clunky to me too, the only required field for `DEBUG` is the ticket path but I don't really want to make all the options not required when they would be in a majority of cases

Reply:
> maybe this belongs in its own module?

Let's keep it in the current module for consistency with other tools